### PR TITLE
Feedback: ViewFeedback component on student homepage

### PR DESCRIPTION
**Pull request by Erin007**
[LP-661](https://codedotorg.atlassian.net/browse/LP-661) - Permanent link to /feedback from the student homepage

New `ViewFeedback` component:
Screen Shot 2019-08-07 at 7 35 57 PM

will display in the Courses section of the student homepage if the user is a student, the `feedbackNotification` experiment is enabled, and the student has feedback: 

student-homepage

### CODE REVIEW

**Review by maddiedierker — APPROVED** [review]
so excited to see this feature coming together!

File: dashboard/app/controllers/home_controller.rb [hunk 1]
```diff
@@ -112,7 +112,9 @@ def init_homepage
       # regular lists of recent scripts.
       exclude_primary_script = true
       @recent_courses = current_user.recent_courses_and_scripts(exclude_primary_script)
-
+      @has_feedback = TeacherFeedback.where(
+        student_id: current_user.id
+      ).count > 0
```
Comment (davidsbailey) [inline dashboard/app/controllers/home_controller.rb:117]: nit: can we skip this query if the user is a teacher? 
```
@has_feedback = current_user.student? && TeacherFeedback.where(...
```

**Review by davidsbailey — APPROVED** [review]
Great work, Erin!

File: dashboard/app/controllers/home_controller.rb (hunk 1 above)
Comment (davidsbailey) [reply dashboard/app/controllers/home_controller.rb:117]: ...or is it theoretically possible for a teacher in another teacher's section to want to see feedback from their teacher?

Comment (Erin007) [reply dashboard/app/controllers/home_controller.rb:117]: It is theoretically possible for a teacher in another teacher's section to want to see feedback from their teacher, but we're only showing this component on the student homepage so I think skipping the query for teachers makes sense here.

**Pull request merged**